functests/5_latency_testing/5_latency_testing_suite_test.go

```diff
@@ -36,7 +36,33 @@ var prePullNamespace = &corev1.Namespace{
 }
 var profile *performancev2.PerformanceProfile
 
-var _ = BeforeSuite(func() {
+func Test5LatencyTesting(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	//TODO Skip the suite before setup steps in case the executable is not found
+
+	setup()
+	defer teardown()
```

Comment:
Are we sure that teardown would run in case of test failure, now that it's not part of the AfterSuite?

Reply:
following the conversation on the team channel: 
defer will not run if an interruption happened during the tests or if an assertion failed. thus, using defer here is not relevant.
Thanks Talor for catching this!